File: frontend/src/app/dto/quiz-material/quiz-material.insert.res.dto.ts
import { QuizMaterialInsertDataResDto } from "./quiz-material.insert.data.res.dto";

export class QuizMaterialInsertResDto{
    private _data! : QuizMaterialInsertDataResDto
	private _message! : string

    public get data(): QuizMaterialInsertDataResDto {
        return this._data;
    }

    public set data(data: QuizMaterialInsertDataResDto) {
        this._data = data;
    }

    public get message(): string {
        return this._message;
    }

    public set message(message: string) {
        this._message = message;
    }

}

File: src/main/java/com/lawencon/elearn/dto/quizmaterial/QuizMaterialInsertResDto.java
package com.lawencon.elearn.dto.quizmaterial;

public class QuizMaterialInsertResDto {
	private QuizMaterialInsertDataResDto data;
	private String message;
	public QuizMaterialInsertDataResDto getData() {
		return data;
	}
	public void setData(QuizMaterialInsertDataResDto data) {
		this.data = data;
	}
	public String getMessage() {
		return message;
	}
	public void setMessage(String message) {
		this.message = message;
	}
}
